templates/fm-overlay-basic.php

```diff
@@ -13,10 +13,14 @@
 }
 
 $fm_overlay_classes = Fm_Overlays_Helpers::instance()->get_overlay_classes( $overlay );
+$targeted_conditions = Fm_Overlays::instance()->targeted_conditions;
 ?>
 
-<!-- @TODO: we should have a dynamic class and/or ID generated for styling purposes -->
-<div id="fm-overlay" class="<?php echo esc_attr( $fm_overlay_classes ) ?>">
+<div
+	id="fm-overlay"
+	class="<?php echo esc_attr( $fm_overlay_classes ) ?>"
+	data-cookiename="<?php echo sanitize_html_class( Fm_Overlays::instance()->get_overlay_cookie_name( $overlay->ID ) ); ?>"
```

Comment:
Why not use `esc_attr()` here since `data-cookiename` is not a list of class names...right?